pkg/skaffold/jib/jib_init.go

```diff
@@ -156,7 +156,11 @@ func ValidateJibConfig(path string) []Jib {
 			return nil
 		}
 
-		results[i] = Jib{BuilderName: builderType.Name(), Image: parsedJSON.Image, FilePath: path, Project: parsedJSON.Project}
+		results[i] = Jib{BuilderName: builderType.Name(), Image: parsedJSON.Image, FilePath: path}
```

Comment:
What happens if a multi-module project has only one sub-module? That is, a root `pom.xml` (`<packaging>pom`) and a single sub-directory for one module. Does `jib:skaffold-init` return one `BEGIN JIB JSON`?

Reply:
I can double check, but I think it returns one for the root project and one for the module.

Reply:
I'm beginning to think this is best fixed Jib-side. I'm going to mess around with Jib and see if I can get somewhere close, and if so then I'll close this.